ansible_collections/arista/avd/roles/eos_designs/templates/facts/switch/switch.j2

```diff
@@ -12,6 +12,14 @@ switch:
   id: {{ switch_id }}
 {% endif %}
 
+{# switch.is_deployed #}
+{% if hostvars[inventory_hostname]['is_deployed'] is arista.avd.defined(false) %}
+{%     set is_deployed = false %}
+{% else %}
+{%     set is_deployed = true %}
+{% endif %}
+  is_deployed: {{ is_deployed }}
+
```

Comment:
```suggestion
{# switch.is_deployed #}
  is_deployed: {{ is_deployed | arista.avd.default(true)

```

Reply:
Will update, can't commit directly as ```}}``` is missing